src/System.IO.Compression/tests/DeflateStreamTests.netstandard1.7.cs

```diff
@@ -0,0 +1,45 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.IO.Compression.Tests
+{
+    public class DeflateStreamTests_NS17 : DeflateStreamTests
```

Comment:
By deriving from DeflateStreamTests, we're going to end up running all of the tests in DeflateStreamTests here again.  That makes sense if those tests use a virtual method for key behaviors and then this derived type overrides that method, but I don't think that's happening here.